packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -87,6 +94,21 @@ const COLUMN_DISK = {
     ),
   name: _('remoteDisk'),
 }
+const COLUMN_SPEED = {
+  itemRenderer: remote => {
+    const benchmark = get(() => remote.benchmarks[remote.benchmarks.length - 1])
+
+    return (
+      benchmark !== undefined && (
+        <span>{`${formatSpeed(benchmark.writeRate, 1e3)} / ${formatSpeed(
+          benchmark.readRate,
+          1e3
+        )}`}</span>
+      )
+    )
+  },
+  name: _('remoteSpeed'),
+}
```

Comment:
This display is not clear enough, a user will not understand what these values are and when they have been measured.

Also they will probably wonder why they don't have the speed during backups.